homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -240,6 +242,25 @@
     ATTR_ILLUMINANCE: "illuminance",
 }
 
+AVAILABLE_ATTRIBUTES_AIRPURIFIER_2H = {
+    ATTR_TEMPERATURE: "temperature",
```

Comment:
The sensor measurements should be separate sensor entities.

Reply:
This change only adds one more model to the already available xiaomi_miio devices. 
I've made no changes to how the purifiers are set up. What I mean is: if there is a need to have separate sensors for the purifier temperature, it should be done for all the already available models, and that is out of the scope of this PR.